customersupport/models.py

```diff
@@ -252,3 +248,101 @@ def replace_date(self):
     @property
     def refund_date(self):
         return self._refund_date
+
+class TicketType(enum.Enum):
+    REFUND="REFUND"
+    REPAIR="REPAIR"
+
+class TicketStatus(enum.Enum):
+    CLOSED="CLOSED"
+    OPEN="OPEN"
+    PENDING="PENDING"
+
+class Ticket(Base):
+    """
+    This class stores tickets in the database.
+
+    Tickets expose the following data
+    Id of the ticket:           id
+    Issue Enumeration:          issue
+    Date Opened:                date_opened
+    Date Closed:                date_closed
+    Current Status Enumeration: current_status
+    List of Call Sessions:      sessions
+    Customer on Ticket:         customer_id
+    Order on Ticket:            order_id
+
+    """
+    __tablename__ = 'ticket'
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    issue = Column(Enum(TicketType))
+    date_opened = Column(DateTime)
+    date_closed = Column(DateTime)
+    current_status = Column(Enum(TicketStatus))
+    sessions = relationship('CallLog',backref="post",cascade="all, delete-orphan",lazy="dynamic")
+    customer_id = Column(String(16)) 
+#    order_id = Column(String(60),ForeignKey('order.id'))
+    
+    def __init__(self, ticket_dict):
+        """
+        Params come from API docs
+        :param ticket_dict: dictionary for init ticket
+        """
+        self.issue = ticket_dict["issue"]
+        self.date_opened = ticket_dict["date_opened"]
+        self.date_closed = ticket_dict["date_closed"]
+        self.current_status = ticket_dict["current_status"]
+        self.sessions = ticket_dict["sessions"]
+        self.customer_id = ticket_dict["customer_id"]
+#        self._order_id = ticket_dict["order_id"]
+
+    def __repr__(self):
+        return '<Ticket %r %r %r %r %r %r>' %(self.id, self.issue, self.date_opened, self.current_status, self.sessions, self.customer_id)
```

Comment:
Another "not a blocker but worth noting" comment:
In Python 3, it's preferred to use the [`.format()` version of string formatting](https://docs.python.org/release/3.6.1/library/functions.html#format) as opposed to the `%` notation, which is ["obsolete and may go away in future versions of Python."](https://docs.python.org/release/3.1.5/library/stdtypes.html#old-string-formatting-operations)